stylus/settings/z-index.styl

```diff
@@ -49,7 +49,7 @@ $drawer-index = 60
 $modal-index = 70
 $modal-footer-index = $modal-index + 1
 $modal-toolbar-index = $modal-footer-index + 1
-$alert-index = 80
+$alert-index = 1400 // z-index for MUI dialogs is 1300, alerts need to be on top
```

Comment:
It seems that we force the z-Index for MUI dialogs here https://github.com/cozy/cozy-ui/blob/master/react/MuiCozyTheme/theme.jsx#L421

It doesn't work anymore?

Reply:
Mmh good point, it seems like a better solution. And now, right now it does not work, I am looking into it, thanks

<img width="212" alt="image" src="https://user-images.githubusercontent.com/465582/114000781-98ec4900-985b-11eb-9786-6cb869cb0b22.png">

Reply:
Fixed :) https://ptbrowne.github.io/cozy-ui/react/#/Content?id=cozydialogs

https://github.com/cozy/cozy-ui/pull/1796/commits/92994264382f66cea16ffe49dc51b892f0837aec